amalgamation/Makefile

```diff
@@ -16,6 +16,9 @@ else
 	DEFS+=-DMSHADOW_USE_SSE=0
 endif
 
+ifndef APPLE
+    export APPLE=0
```

Comment:
I'm not familiar with amalgamation and its build in mxnet. Could you point me to where this is used?